mlir/lib/Dialect/Affine/Transforms/LoopTiling.cpp

```diff
@@ -237,6 +237,13 @@ void LoopTiling::getTileSizes(ArrayRef<AffineForOp> band,
 }
 
 void LoopTiling::runOnOperation() {
+  if (cacheSizeInKiB <= 0) {
+    Builder b = Builder(getOperation()->getContext());
+    emitError(b.getUnknownLoc(),
+              "Illegal Argument: '--affine-loop-tile=cache-size' cannot be "
+              "less than or equal to zero. LoopTiling Pass Aborted!");
```

Comment:
Emit messages should start in lower case. Avoid referring to the pass name here - it may change later.